pkg/client/unversioned/request.go

```diff
@@ -244,11 +244,10 @@ func (r *Request) AbsPath(segments ...string) *Request {
 	if r.err != nil {
 		return r
 	}
-	if len(segments) == 1 {
+	r.path = path.Join(r.baseURL.Path, path.Join(segments...))
```

Comment:
on line 127: r.baseURL.Path includes whatever proxy-prefix was specified. no /api, /apis, or version info. this is a semantic change that I've introduced.

on line 128: r.path defaults to versionedAPIPath, which includes the proxy prefix as well as (optionally) a path to an API version (e.g. /api/v1). it's a "fully-qualified path" that's used to access API resources. i haven't changed the semantics of this variable, it's always been the full path to an API variant.

here (247): we're saying "build a path that begins with the proxy-prefix (if there is one) append segments to it"; there's no duplication. This is change from the prior behavior that would wipe out any desired proxy-prefix.